src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -672,11 +678,28 @@ export class InlineChatWidget {
 			return;
 		}
 
-		this._elements.infoLabel.classList.toggle('hidden', false);
-		const label = localize('slashCommandUsed', "Using {0} to generate response...", `\`\`/${details.command}\`\``);
+		this._elements.detectedIntent.classList.toggle('hidden', false);
+
+		this._slashCommandUsedDisposables.clear();
+
+		const label = localize('slashCommandUsed', "Using {0} to generate response. [[Re-run without]]", `\`\`/${details.command}\`\``);
```

Comment:
a complete sentense might be better like "Using /foo to generate reponse (re-run without)."